### PR TITLE
fix: some typos

### PR DESCRIPTION
**Pull request by mo1ein**
```n``` should be capital (```N```) in ```[y/n]``` message, because ```n``` is a default choice =)

### CODE REVIEW

File: pcrypt.py [hunk 1]
```diff
@@ -216,7 +216,7 @@ def read_password(show_password_in_prompt):
 
     def should_we_stop():
         return input(
-            '{white}Continue? [{yellow}y{reset}{white}/{reset}{yellow}n{reset}{white}]: {reset}'
+            '{white}Continue? [{yellow}y{reset}{white}/{reset}{yellow}N{reset}{white}]: {reset}'
```
Comment (pouriya) [inline pcrypt.py:219]: Hmmmm. So the `y` should be `Y` too. Am I right?

File: pcrypt.py [hunk 2]
```diff
@@ -269,7 +269,7 @@ def main(args, tmp_tar_filename):
         default=getcwd(),
         dest='directory',
         help='In decryption mode it should be directory name that contains the encrypted file and' \
-        'in encryption mode it should be the directory that you want to encrypt all or one of its' \
+        ' in encryption mode it should be the directory that you want to encrypt all or one of its' \
```
Comment (pouriya) [inline pcrypt.py]: Thanks! But you have reached max line length (which is 100 characters), So you have to move one character at the end of line to the next line.

**Review by pouriya — CHANGES_REQUESTED** [review]
Also in [line 125](https://github.com/pouriya/pcrypt/blob/6cfa7d6f3852a287a32d195ad0b2c8c9ea8661ea/pcrypt.py#L125) the term `encrypting` should be `decrypting `. Could you fix that too?

File: pcrypt.py (hunk 2 above)
Comment (mo1ein) [reply pcrypt.py]: Yes. I fixed it.

File: pcrypt.py (hunk 1 above)
Comment (mo1ein) [reply pcrypt.py:219]: ```y``` is not default. When we type nothing or everything except ```y``` that means no (```n```) so ```y```(not Capital) is correct.
I mean like this ```[y/N]```.

Comment (pouriya) [reply pcrypt.py:219]: I see. Thank you for clarifying.

**Comment by mo1ein** [comment]
> Also in [line 125](https://github.com/pouriya/pcrypt/blob/6cfa7d6f3852a287a32d195ad0b2c8c9ea8661ea/pcrypt.py#L125) the term `encrypting` should be `decrypting `. Could you fix that too?

Done :heavy_check_mark:.

**Comment by pouriya** [comment]
Thanks a lot.

File: pcrypt.py (hunk 1 above)
Comment (mo1ein) [reply pcrypt.py:219]: You're welcome :blush:.

**Pull request merged**